views/admin.ejs

```diff
@@ -13,17 +13,51 @@
 
     <main>
       <div>
-        <label>Admin Link: </label>
-        Replace with actual link!!
+        <label>Admin Link:   </label>
+        <a><%=admin_link%></a>
       </div>
       <div>
-        <label>Voter Link: </label>
-        Replace with actual link!!
+        <label>Voter Link:   </label>
+        <%=voter_link%>
       </div>
-      use for loop or sth to generate results list
       <div>
-        <button type="submit" class="btn" id="refresh">Refresh Results</button>
-        <button type="submit" class="btn" id="close">Close Poll</button>
+        <label>Title:   </label>
+        <%=title%>
+      </div>
+      <div>
+        <label>Description:   </label>
+          <%=description%>
+        </div>
+      <div class = "result_choice_score">
+        <table class="table">
+          <tbody>
+            <% for ( let i of choicesandvalue) { %>
+              <tr>
```

Comment:
If we are keeping the master version (which I suggest) the template vars will need to be changed here.